### PR TITLE
Adding Octave package to BF downloads page (rebased onto dev_5_1)

### PR DESCRIPTION
**Pull request by hflynn**
This is the same as gh-178 but rebased onto dev_5_1.

---

Re: https://github.com/openmicroscopy/ome-release/issues - adds Octave package download to the Bio-Formats tools downloads table.

Docs added in https://github.com/openmicroscopy/bioformats/pull/2200

### CODE REVIEW

**Comment by hflynn** [comment]
--rebased-from #178

File: bf_downloads.html [hunk 1]
```diff
@@ -123,6 +123,15 @@
                                 <td align="center">@MATLAB_TOOLS_SHA1@ (<a
                                         href="@MATLAB_TOOLS@.sha1">SHA1</a>)</td>
                             </tr>
+                            <tr>
+                                <td><a href="@OCTAVE_PACKAGE@"><img
+                                        src="images/bf_octave_package.png"
+                                        alt="Octave Package" /></a></td>
+                                <td align="center">@OCTAVE_PACKAGE_SIZE@</td>
+                                <td>@OCTAVE_PACKAGE_BASE@</td>
+                                <td align="center">OCTAVE_PACKAGE_SHA1@ (<a
```
Comment (sbesson) [inline bf_downloads.html]: There is a missing `@` before `OCTAVE_PACKAGE_SHA1@` which prevents the token to be replaced

**Comment by hflynn** [comment]
Okay, should actually be fixed now!

**Comment by sbesson** [comment]
:+1: rebuilt and tested by @bramalingam

**Pull request merged**